.coveragerc

```diff
@@ -1,3 +1,13 @@
 [run]
 source = openedx_webhooks
 branch = true
+omit =
+    */conftest.py
+    */test_*.py
+    openedx_webhooks/github/dispatcher/rules/github_activity.py
+    openedx_webhooks/github/models.py
+    openedx_webhooks/jira/tasks.py
+    openedx_webhooks/lib/edx_repo_tools_data/utils.py
+    openedx_webhooks/lib/jira/client.py
+    openedx_webhooks/webhook_confs.py
+    openedx_webhooks/worker.py
```

Comment:
I'm skeptical that we have code that is important to have, but uninteresting to test.  Excluding code from test coverage seems like a dangerous precedent.